packages/driver/cypress/e2e/commands/aliasing.cy.js

```diff
@@ -34,11 +26,13 @@ describe('src/cy/commands/aliasing', () => {
       cy.get('@body')
     })
 
-    it('stores the resulting subject as the alias', () => {
-      const $body = cy.$$('body')
-
+    it('stores the resulting subject chain as the alias', () => {
       cy.get('body').as('b').then(() => {
-        expect(cy.state('aliases').b.subject.get(0)).to.eq($body.get(0))
+        const { subjectChain } = cy.state('aliases').b
+
+        expect(subjectChain.length).to.eql(2)
+        expect(subjectChain[0]).to.be.undefined
```

Comment:
Why is the first subject chain undefined? Shouldn't the only subject chain be `.get()`? Shouldn't we assert the second subject chain is actually `.get()`?

Reply:
For the second part - yes, yes we should assert that the second entry is `.get()`. Added an assertion.